### DISCUSSION TITLE
help me find SeuratDisk

### DISCUSSION THREAD

**Discussion by genesNi** [satijalab/seurat#9280] [Q&A] [answered]
in R.4.4.1 version SeuratDisk is not available where do I search for it

**Comment by samuel-marsh** [comment, marked as answer]
Hi SeuratDisk can be found on Github (https://github.com/mojaveazure/seurat-disk).  However, please not that devs have stated the SeuratDisk is no longer being maintained and may not function correctly with current Seurat object formats or formats of other objects (anndata etc).

Best,
Sam
